{{cookiecutter.project_slug}}/apps/settings/models.py

```diff
@@ -39,3 +40,47 @@ class FooterHTML(BaseSetting):
 
     class Meta:
         verbose_name = "HTML - Footer"
+
+
+@register_setting(icon="link")
+class SocialAccounts(BaseSetting):
+    facebook_url = models.URLField(
+        blank=True,
+        help_text="""
+        The URL of your organisation\'s Facebook page. E.g.
```

Comment:
This shouldn't need to be escaped in multi-line string

Reply:
do you mean it should be one line, or just using apostrophes and not speech marks?

Reply:
looked into this one, its too long to go on one